android/build.gradle

```diff
@@ -77,7 +77,7 @@ android {
     }
 
     dependencies {
-        implementation 'com.google.android.libraries.navigation:navigation:5.1.1'
+        implementation 'com.google.android.libraries.navigation:navigation:6.0.0'
```

Comment:
@illuminati1911 - did we test the sample apps locally? IIRC, bumping to Nav 6.0 requires updating KT/Java which is why we originally suggested going to 5.99.0 for now.

Reply:
@ArturoSalazarB16 Yeah, the sample app builds and runs without any issue.